airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_error_reporter/SentryExceptionHelper.java

```diff
@@ -166,4 +169,43 @@ private static Optional<List<SentryException>> buildJavaSentryExceptions(final S
     return Optional.of(sentryExceptions);
   }
 
+  private static Optional<List<SentryException>> buildNormalizationDbtSentryExceptions(final String stacktrace) {
```

Comment:
how common is it that the line immediately following `Database error` is the "interesting" one? e.g: is it 100% of the time based on a reasonable empirical sample size?

Reply:
So I was going for a classic 80-20 coverage move here, using the following data:

[all normalization-system-errors (with the new failure reasons)](https://airbyte.metabaseapp.com/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJxdWVyeSIsInF1ZXJ5Ijp7InNvdXJjZS10YWJsZSI6MTAsImZpbHRlciI6WyJhbmQiLFsiPSIsWyJmaWVsZCIsMjQ0NzAsbnVsbF0sIm5vcm1hbGl6YXRpb24tc3lzdGVtX2Vycm9yIl1dfSwiZGF0YWJhc2UiOjJ9LCJkaXNwbGF5IjoidGFibGUiLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7InRhYmxlLmNvbHVtbl93aWR0aHMiOltudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCwxMzc4LG51bGwsbnVsbCxudWxsLG51bGwsMjAyLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCw0OTBdfX0=)
1,110 rows

[Filtering for 'database errors'](https://airbyte.metabaseapp.com/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoyLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjEwLCJmaWx0ZXIiOlsiYW5kIixbIj0iLFsiZmllbGQiLDI0NDcwLG51bGxdLCJub3JtYWxpemF0aW9uLXN5c3RlbV9lcnJvciJdLFsiY29udGFpbnMiLFsiZmllbGQiLDI0NDcyLG51bGxdLCJEYXRhYmFzZSBFcnJvciBpbiBtb2RlbCIseyJjYXNlLXNlbnNpdGl2ZSI6ZmFsc2V9XV19LCJ0eXBlIjoicXVlcnkifSwiZGlzcGxheSI6InRhYmxlIiwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6eyJ0YWJsZS5jb2x1bW5fd2lkdGhzIjpbbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsMTM3OCxudWxsLG51bGwsbnVsbCxudWxsLDIwMixudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsNDkwXX19)
736 rows
Nominal 736/1110 (66%)
Unique 368/442 (83%)

694/736 (~94%) follow the pattern of line immediately following "Database Error in model" is the single useful part.

Reply:
Did some futher investigation today and decided it's not significant extra work to cover a lot of the possible error structures based on what we've seen so far:

The other 42 errors follow 1 of 3 patterns:
- SQL compilation error:\n (next line with detail)
- Invalid input (a line X after this one that is "context: {}" has detail)
- syntax error at or near "{value}" (next line with detail)

Currently implementing specific logic for these three extra edge cases so we cover all currently known database errors. (No doubt there are other edge cases that exist and we haven't encountered but we can build upon this parsing code over time.)

Also implementing logic for the rest of the error types (the other ~20% of unique errors we've seen):
- Unhandled error
- Compilation Error
- Runtime Error

Basing the logic from our dataset in Metabase linked above, now covering >95% of every dbt error we've seen since implementing failure reasons.

Reply:
> is this the title we see in the list of incidents in sentry's incident view? if so could we change that to be more unique for each issue?
These changes will also solve this point